R/copy_SS_inputs.R

```diff
@@ -135,12 +135,21 @@ copy_SS_inputs <- function(dir.old = NULL,
       dir.exe <- dir.old
     }
     # figure out which files are executables
-    is.exe <- file.info(dir(dir.exe, full.names = TRUE))$exe
-    exefiles <- dir(dir.exe)[is.exe != "no"]
-    if (.Platform[["OS.type"]] == "unix") {
+    if (.Platform[["OS.type"]] == "windows") {
+      # "exe" column in output from file.info() only on windows
+      is.exe <- file.info(dir(dir.exe, full.names = TRUE))[["exe"]]
+      exefiles <- dir(dir.exe)[is.exe != "no"]
+    } else {
       # Anything without an extension is listed (may need to make this more
       # specific??)
-      exefiles <- list.files(dir.exe, pattern = "^[^.]+$")
+      # exefiles <- list.files(dir.exe, pattern = "^[^.]+$")
+
+      # directories are accidentally being included using line above,
+      # so trying a different way to exclude them
+      exefiles <- dir(dir.exe)[!file.info(dir(dir.exe,
+        full.names = TRUE
+      ))[["isdir"]]]
+      exefiles <- grep(pattern = "^[^.]+$", x = exefiles, value = TRUE)
       message("Unix binaries are: ", paste0(exefiles, collapse = ", "))
```

Comment:
maybe wrap this message in `if (verbose)`?

Reply:
Doen.